app/api/v1/models.py

```diff
@@ -0,0 +1,125 @@
+# This is our models file where we store our temporary storage data and methods
+# to manipulate the data. Our parcels list is globally available all classes
+# that inherit the Parcel class
+
+# this is where all our parcels will be appended
+parcels = [
+    {
+        "id": 1,
```

Comment:
Kindly be persistent when using double quotes. I noticed you mix both single and double quotes, so please use one of them in every file.